README.md

```diff
@@ -150,7 +150,7 @@ Run `pod install` in your `ios/` directory.
 <br>
 Create a bridging header by right clicking on your project folder:
 
-- Selecting Add Files to "PROJECT_NAME"
+- Selecting New File...
 - Choose Swift
 - Make sure the selected target is your main app target
 - Create file and say yes to creating a Bridging Header file
```

Comment:
```suggestion
- Create a file and say yes to creating a Bridging Header file
```

Reply:
Thanks! Reworded a bit to sound more natural